tests/functional/api/BackbeatServer.js

```diff
@@ -794,8 +796,8 @@ describe('Backbeat Server', () => {
                 ];
                 setKey(redisClient, keys, err => {
                     assert.ifError(err);
-                    const route =
-                        '/_/crr/failed/test-bucket/test-key/test-versionId';
+                    const route = '/_/crr/failed?bucket=test-bucket&' +
```

Comment:
Is there a need to pass bucket & key as query params as well? If not necessary I would recommend to stick with AWS SDK way of doing GETs on objects to avoid adding new conventions when not necessary (which would give `/_/crr/failed/bucket/object?versionId=xyz`)

Reply:
It's true that there's no need to pass the bucket (since it cannot contain the char `/`). I can update the route to `/_/crr/failed/bucket?key=abc&versionId=xyz`.

Reply:
Updated.